frontend/src/Login/ForgottenPassword.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import {
+  View, StyleSheet, Text,
+  ImageBackground, Alert,
+} from 'react-native';
+import { useNavigation } from 'react-navigation-hooks';
+import Colors from '../common/colors';
+import background from '../../assets/login/background.jpg';
+import InputField from './InputField';
+import SubmitButton from './SubmitButton';
+
+const styles = StyleSheet.create({
+  container: {
+    alignItems: 'center',
+    justifyContent: 'center',
+    padding: 0,
+  },
+  header: {
+    width: 300,
+    marginVertical: 12,
+    fontSize: 24,
+    fontWeight: 'bold',
+    color: Colors.whiteColor,
+  },
+  background: {
+    width: '100%',
+    height: '100%',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  button: {
+    flexDirection: 'row',
+    marginTop: 20,
+    justifyContent: 'space-between',
+  },
+});
+
+function showAlert(text) {
+  Alert.alert('Warning', text);
+}
+
+function ForgottenPassword() {
+  const navigation = useNavigation();
+  const [input, setInput] = useState('');
+
+  function handleSubmit() {
+    if (input === '') {
+      return showAlert('The input field is required.');
+    }
+    if (input.length <= 3) {
+      return showAlert('Please enter a long username.');
+    }
+    return navigation.pop();
+  }
+
+  return (
+    <View style={styles.container}>
+      <ImageBackground
+        style={styles.background}
+        resizeMode="cover"
+        source={background}
+      >
+        <Text style={styles.header}>Forget Password</Text>
+        <View style={{ width: 300 }}>
+          <InputField
+            placeholder="input"
```

Comment:
I think our users may not know what data should input.
placeholder name could be username or email.